src/app/screens/ConfirmPayment/index.tsx

```diff
@@ -82,7 +84,13 @@ function ConfirmPayment() {
       );
       auth.fetchAccountInfo(); // Update balance.
       msg.reply(response);
-      setSuccessMessage(t("success"));
+      setSuccessMessage(
+        t("success", {
+          amount: `${invoice.satoshis} (SATS)${
```

Comment:
No brackets neededaround the sats :)  
https://github.com/getAlby/lightning-browser-extension/issues/1574